string varName
float64 value
